lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class DSLocationMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final DSMessageBubbleStyle style;
+  final String? title;
+  final String latitude;
+  final String longitude;
+
+  DSLocationMessageBubble({
+    super.key,
+    required this.align,
+    required this.latitude,
+    required this.longitude,
+    DSMessageBubbleStyle? style,
+    this.title,
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  final appKey = 'AIzaSyAlC3a3DZZBscR0QIbQpee13Op9Y05m_wc';
+
+  @override
+  Widget build(BuildContext context) {
+    final foregroundColor = style.isLightBubbleBackground(align)
+        ? DSColors.neutralDarkCity
+        : DSColors.neutralLightSnow;
+
+    return GestureDetector(
+      onTap: () => launchUrl(
+        Uri.parse(
+          'https://www.google.com/maps/search/?api=1&query=$latitude,$longitude',
+        ),
+      ),
+      child: DSMessageBubble(
+        shouldUseDefaultSize: true,
+        defaultMaxSize: 400.0,
+        defaultMinSize: 400.0,
+        padding: EdgeInsets.zero,
+        align: align,
+        style: style,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            DSCachedNetworkImageView(
+              url:
+                  'https://maps.googleapis.com/maps/api/staticmap?&size=400x300&markers=$latitude,$longitude&key=$appKey',
```

Comment:
You should change the size to 240px by changing its query parameter.

```suggestion
                  'https://maps.googleapis.com/maps/api/staticmap?&size=240x240&markers=$latitude,$longitude&key=$appKey',
```